srcts/src/bindings/input/text.ts

```diff
@@ -22,15 +22,16 @@ function getLabelNode(el: HTMLElement): JQuery<HTMLElement> {
 
 class TextInputBindingBase extends InputBinding {
   find(scope: HTMLElement): JQuery<HTMLElement> {
-    const $inputs = $(scope).find(
+    const matches = $(scope).find(
```

Comment:
Just for sake of a minimal diff

```suggestion
    const $inputs = $(scope).find(
```